build.gradle

```diff
@@ -26,7 +26,7 @@ ext {
   javaVersion = JavaVersion.VERSION_1_7
 
   GROUP = 'com.squareup.leakcanary'
-  VERSION_NAME = "1.4-SNAPSHOT"
+  VERSION_NAME = "1.4-beta2"
```

Comment:
TL;DR - that'd be the next commit.

I'm rather fond of how Retrofit and OkHttp do it, i.e., the "prepare for next development iteration" commit which winds up being the 1st commit after the previous release's tag.  If you compare tag...master, the "iteration prep" commit appears first, "release prep" commits last.